src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -209,6 +212,19 @@ public DatasetModel retrieveDatasetModel(
   public DatasetModel retrieveDatasetModel(UUID id, AuthenticatedUserRequest userRequest) {
     return retrieveDatasetModel(id, userRequest, getDefaultIncludes());
   }
+  /**
+   * Helper method to retrieve the required actions to view the dataset retrieve the dataset a
```

Comment:
Is "retrieve the dataset a second time if you already have it" meant to be in this docstring?